tests/fs_diff/test_fs_diff.py

```diff
@@ -0,0 +1,25 @@
+import os
+import pytest
+import subprocess
+
+# Requirements:
+# - 2 XCP-ng host of same version
+
+pytest.fixture(scope='module')
+def test_fs_diff(hosts):
+    assert len(hosts) == 2, "This test requires exactly 2 hosts"
+
+    assert (hosts[0].xcp_version == hosts[1].xcp_version), f"Hosts versions differ"
```

Comment:
The message associated to an assert should state what is expected, not what's wrong when it fails.

Reply:
Fixed